tests/storage/storage_migration/test_mtc_storage_class_migration.py

```diff
@@ -242,7 +242,7 @@ def test_vm_storage_class_migration_windows_vm_with_vtpm(
         vms_boot_time_before_storage_migration,
         storage_mig_plan,
         storage_mig_migration,
-        deleted_old_dvs_of_online_vms,
+        # deleted_old_dvs_of_online_vms,
```

Comment:
So we need to keep the `deleted_old_dvs_of_online_vms`, 
but remove `deleted_completed_virt_launcher_source_pod` from `deleted_old_dvs_of_online_vms`